repositories.bzl

```diff
@@ -572,14 +574,25 @@ cc_proto_library(
         "//external:cc_wkt_protos",
     ],
 )
-""".format(protobuf_repo)
+"""
 
-    # Update the SHA due to backend.proto updates, to support
-    # backend routing.
+    # googleapis currently require bazel version >= 23.0.
+    # BUILD files were added in the googleapis repository in October 2018, causes "crosses boundary
+    # of subpackage error". As a workaround, a forked repo without BUILD files is used.
+    #
+    # Instructions for updating googleapis:
+    #  1) fork from https://github.com/googleapis/googleapis.git
+    #  2) make a branch
+    #  3) remove BUILD files `$find . -type f -name '*BUILD*' | xargs rm`
+    #  4) commit and push
+    #  5) Update `commit` and `remote` in `new_git_repository`
+    #  6) Update the below comment to reflect the new version
+    #
+    # Corresponds to 9a02c5acecb43f38fae4fa52c6420f21c335b888 of googleapis/googleapis.git
```

Comment:
also specify the date for this SHA

Reply:
The date is below. I can move it up if you think that's better.

Reply:
It is not clear. I though the data below is for the fork SHA. maybe move it up is better.